src/libs/actions/Wallet.js

```diff
@@ -528,6 +528,42 @@ function updateCurrentStep(currentStep) {
     Onyx.merge(ONYXKEYS.USER_WALLET, {currentStep});
 }
 
+/**
+ * @param {Object} answers
```

Comment:
Shouldn't this be `@param {Array}`? (based on https://github.com/Expensify/App/blob/1c78eb83004e76d77afde68186410d3118293b65/src/pages/EnablePayments/IdologyQuestions.js#L52-L53)

Reply:
You're right! This param is an array. Updated!

<img width="936" alt="Screen Shot 2022-09-15 at 2 43 42 PM" src="https://user-images.githubusercontent.com/12268372/190426031-92c2dc8e-bd3b-4ce1-85ec-7d0794644eac.png">